plugin/route53/README.md

```diff
@@ -16,6 +16,7 @@ The route53 plugin can be used when coredns is deployed on AWS or elsewhere.
 route53 [ZONE:HOSTED_ZONE_ID...] {
     [aws_access_key AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY]
     upstream [ADDRESS...]
+    [credentials_file PROFILE FILENAME]
```

Comment:
just put it like :`credential PROFILE [FILENAME]`

that the entire option is optionally and require `[` and `]` is not something we do consistently, also introduces quite some noise in the docs IMO